src/tooling/migration-rrule-recurrence/migration-rrule-recurrence.js

```
@@ -63,47 +79,82 @@ async function update(collectionName, documents) {
+                    rRule.bymonth = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+                }
+            } else {
+                rRule.bymonth = [];
+            }
+        }
+
+        const queryForDelete = {_id: card._id};
+        let replacementCard = card;
+
+        if (Object.keys(rRule).length) {
+            delete replacementCard.timeSpans;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)